### DISCUSSION TITLE
Add speed to unit system?

### DISCUSSION THREAD

**Discussion by rianadon** [home-assistant/architecture#588] [General] [answered]
Right now, the unit system covers temperature, length, pressure, and volume.

However, weather entities use speed units for measuring wind speed, and exercise equipment / car integrations might use speed as well. Since speed isn't included in the unit system, it seems most integrations do the conversion manually, using the distance conversion methods to convert between miles/hour and kilometers/hour.

Would it make since, then, to make speed part of the unit system?

**Comment by MartinHjelmare** [comment, marked as answer]
We have conversions now for speed in our unit utils.
